package.json

```diff
@@ -106,6 +106,45 @@
             "isSticky": true,
             "description": "Analyze a collection's schema."
           }
+        ],
+        "disambiguation": [
+          {
+            "category": "mongodb",
+            "description": "The user is asking for assistance or information related to MongoDB.",
+            "examples": [
+              "Explain this MongoDB query.",
+              "How do I write this MongoDB query in Python?",
+              "How do I use $lookup in MongoDB?",
+              "Can you help me debug a MongoDB query?",
+              "Rewrite this MongoDB query",
+              "How to convert this MongoDB query to Python?",
+              "How to convert this MongoDB query to Java?",
+              "How to convert this MongoDB query to Node.js?",
+              "How to convert this MongoDB query to Go?",
+              "How to convert this MongoDB query to php?",
+              "How do I update documents in MongoDB?",
+              "How do I insert documents in MongoDB?",
+              "How do I create documents in MongoDB?",
+              "How do I delete documents in MongoDB?",
+              "Explain how to use MongoDB transactions.",
+              "Help me write this MongoDB query:",
+              "Explain MongoDB query syntax.",
+              "Help me write a MongoDB aggregation pipeline.",
+              "Help me debug this aggregation pipeline.",
+              "How do I use $lookup in a MongoDB aggregation pipeline?",
+              "How do I optimize this MongoDB query?",
+              "How do I optimize indexes in MongoDB?",
+              "How do I optimize this data model in MongoDB?",
+              "How do I create a compound index for this query?",
+              "How do I model relationships in MongoDB?",
+              "MongoDB best practices for indexing",
+              "How do I use MongoDB Atlas search?",
+              "How do I perform semantic search in MongoDB?",
+              "How do I stream data using MongoDB?",
+              "How do I monitor change streams in MongoDB?",
+              "How do I set up a MongoDB replica set for local development?"
+            ]
```

Comment:
Should we add some examlers about schema?

Reply:
Added some more examples about schema!